src/assets/styles/app.css

```diff
@@ -175,9 +175,11 @@ main {
 main header h1 {
     font-size: 3rem;
     font-weight: 800;
+    hyphens: auto;
```

Comment:
Hyphens and word breaks probably should be applied to all headings and content because long words could appear anywhere. If you set UIO text size to 2 and switch to iphone 5 view, the home page is still horizontally scrollable because there are a few long words in h2 and contents.

Reply:
This is an excellent observation, and something I noticed as well though I wasn't sure if I should consider it in scope for this issue. I will add this rule :)